test/utils.sh

```diff
@@ -405,20 +425,53 @@ ocp_to_opm_version_mapping() {
 # It returns the ocp_version targeted by the FBC fragment
 get_ocp_version_from_fbc_fragment() {
   local FBC_FRAGMENT="$1"
+  local ocp_version_list
+  local ocp_version_list_raw
 
   if [ -z "$FBC_FRAGMENT" ]; then
     echo "get_ocp_version_from_fbc_fragment: Missing positional parameter \$1 (FBC_FRAGMENT)" >&2
     exit 2
   fi
 
-  local base_image
-  base_image=$(get_image_registry_repository_tag "$(get_base_image "$FBC_FRAGMENT")")
-  ocp_version=$(parse_image_url "$base_image" | jq -ej '.tag')
-  if [ -z "${ocp_version}" ]; then
-    echo "get_ocp_version_from_fbc_fragment: No ocp version found; base image tag is empty." >&2
-    exit 2
+  # Attempt to retrieve the ocp version list from the fragment labels
+  ocp_version_list_raw=$(
+    get_image_labels "$FBC_FRAGMENT" \
+      | grep -m1 '^com.redhat.fbc.openshift.version='
+  )
+
+  # Check if the label is present within labels (If yes, check content, if not use the base image ocp version)
+  if [ "$ocp_version_list_raw" ]; then
```

Comment:
Consider changing this to check if the string length is non-zero

```
if [[ -n "$ocp_version_list_raw" ]]; then
```

Reply:
updated